conf/char-server.conf

```diff
@@ -95,8 +95,8 @@ autosave_time: 60
 save_log: yes
 
 // Start point, Map name followed by coordinates (x,y)
-start_point: iz_int,97,90
-start_point_pre: new_1-1,53,111
+start_point_renewal: iz_int,97,90
+start_point_pre-re: new_1-1,53,111
```

Comment:
Can we stick to letters, numbers and underscores, for config entries? That way it'll cause less problems when we switch to libconfig (ongoing branch)